README.md

```diff
@@ -29,57 +26,43 @@ We appreciate your feedback and energy helping us improve our services. [Please
-    - ./Chat/ClientApp/src/Core : Containers a redux wrapper around the Chat SDK
-    - ./Chat/ClientApp/src/index.js : Entry point for the client app
-- ./Chat/Controllers : Server app core logic for client app to get a token to use with the Azure Communication Services Web Chat SDK
-- ./Chat/Program.cs : Server app program logic
-- ./Chat/Startup.cs: Server app startup logic
+- ./Chat/src/app: Where the client code lives
+- ./Chat/src/app/App.tsx: Entry point into the chat sample 
+- ./Chat/src/app/HomeScreen.tsx: The first screen in the chat sample
+- ./Chat/src/app/ConfigurationScreen.tsx: Where to set your name and avatar for the chat
+- ./Chat/src/app/ChatScreen.tsx: The main chat thread screen
+- ./Chat/src/app/EndScreen.tsx: The screen the user will see when they are no longer in the chat thread
+- ./Chat/src/app/ErrorScreen.tsx: The screen when an error occurs in the chat thread
+- ./Server: server code
+- ./Server/appsettings.json: Where to put your azure communication services connection string
 
 ## Before running the sample for the first time
 
 1. Open an instance of PowerShell, Windows Terminal, Command Prompt or equivalent and navigate to the directory that you'd like to clone the sample to.
 2. `git clone https://github.com/Azure-Samples/communication-services-web-chat-hero.git`
 3. Get the `Connection String` from the Azure portal. For more information on connection strings, see [Create an Azure Communication Resources](https://docs.microsoft.com/en-us/azure/communication-services/quickstarts/create-communication-resource)
-4. Once you get the `Connection String`, Add the connection string to the **Chat/appsettings.json** file found under the Chat folder. Input your connection string in the variable: `ResourceConnectionString`.
+4. Once you get the `Connection String`, Add the connection string to the **Server/appsettings.json** file found under the Chat folder. Input your connection string in the variable: `ResourceConnectionString`.
 
 ## Local run
 
-1. Go to the Chat folder and open the `Chat.csproj` solution in Visual Studio
-2. Run the project. The browser will open at localhost:5000.
-
-### Troubleshooting
-
-1. Solution doesn\'t build, it throws errors during NPM installation/build
-
-    Clean/rebuild the C# solution
+1. Set your connection string in `Server/appsettings.json`
+1. `npm run setup` from the root directory
+2. `npm run start` from the root directory
 
 ## Publish to Azure
 
-1. Right click the `Chat` project and select Publish
-2. Create a new publish profile and select your app name, Azure subscription, resource group and etc.
-3. Before publish, add your connection string with `Edit App Service Settings`, and fill in `ResourceConnectionString` as key and connection string (copy from appsettings.json) as value
-
-## Building off of the sample
-
-If you would like to build off of this sample to add chat capabilities to your own awesome application, keep a few things in mind:
```

Comment:
small addition (that I'm leaving out but mentioning it here in case) - eslint we tell them to add `// Copyright (c) Microsoft Corporation.` to all of their files